webpack/global_index.js

```diff
@@ -7,8 +7,11 @@ import RegistrationCommands from './components/extensions/RegistrationCommands';
 import ContentTab from './components/extensions/HostDetails/Tabs/ContentTab';
```

Comment:
Renamed this from `global_fills.js` to `global_index.js` as this is now the best practice according to the Foreman plugin template.